source/addonHandler/__init__.py

```diff
@@ -422,7 +422,7 @@ def isPendingDisable(self):
 
 	def _getPathForInclusionInPackage(self, package):
 		extension_path = os.path.join(self.path, package.__name__)

```

Comment:
I think it would be best to fetch package directly name directly instead of going through this function unless someone says otherwise. I'm okay with this at the moment, although I think we may at some point revisit this function.